blivet/blivet.py

```diff
@@ -812,7 +812,10 @@ def create_device(self, device):
             :rtype: None
         """
         self.devicetree.actions.add(ActionCreateDevice(device))
-        if device.format.type and not device.format_immutable:
+
+        is_snapshot = isinstance(LVMLogicalVolumeDevice) and device.is_snapshot_lv
```

Comment:
``isinstance()`` takes two arguments (which is what pylint is complaining about). Other than that this looks good to me.